ocaml/xcp-rrdd/bin/rrdp-iostat/rrdp_iostat.ml

```diff
@@ -232,41 +232,51 @@ module Stat = struct
   type t = int64 list
   (* /sys/block/tdX/stats @ /sys/block/tdX/inflight @ [read bytes; write bytes] *)
 
+  let finally = Xapi_stdext_pervasives.Pervasiveext.finally
+
+  (** open [path] to let [f] scan the contents using scanf *)
+  let scanning path f =
+    let io = Scanf.Scanning.open_in path in
```

Comment:
Would ic be a better name as it's an input_channel, not input/output?

Reply:
Maybe. But this is a 3-line function so I don't see room for confusion and `io` is a stronger mnemonic than `ic`. I've used `io` in other parts of the code - writing to it would be detected by the type system.